src/main/java/uk/gov/homeoffice/digital/sas/person/enums/TermsAndConditions.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.homeoffice.digital.sas.person.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum TermsAndConditions {
+  MODERNISED("MODERNISED"),
+  PRE_MODERNISED("PRE_MODERNISED");
```

Comment:
The value doesn't look like it's required, typically it would be a 'human' readable version e.g. 'Pre modernised' or none at all. 
We should be able to get what's currently in `stringValue` by calling the `name()` method.

Reply:
Sure, Way too much code for no reason.

Reply:
@sewerynmi make sure the API still works after this change, just in case :)

Reply:
@sewerynmi feel free to resolve this yourself based on @taomoh comment